internal/blockprocessor/index_entries_test.go

```diff
@@ -0,0 +1,812 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package blockprocessor
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/IBM-Blockchain/bcdb-server/internal/worldstate"
+	"github.com/IBM-Blockchain/bcdb-server/internal/worldstate/leveldb"
+	"github.com/IBM-Blockchain/bcdb-server/pkg/logger"
+	"github.com/IBM-Blockchain/bcdb-server/pkg/types"
+	"github.com/stretchr/testify/require"
+)
+
+type indexTestEnv struct {
+	db      *leveldb.LevelDB
+	dbPath  string
+	cleanup func()
+}
+
+func newIndexTestEnv(t *testing.T) *committerTestEnv {
+	lc := &logger.Config{
+		Level:         "debug",
+		OutputPath:    []string{"stdout"},
+		ErrOutputPath: []string{"stderr"},
+		Encoding:      "console",
+	}
+	logger, err := logger.New(lc)
+	require.NoError(t, err)
+
+	dir, err := ioutil.TempDir("/tmp", "index")
```

Comment:
The empty string "" instead of "/tmp" makes this platform independent

Reply:
Done